docs/hub/model-repos.md

```diff
@@ -293,6 +293,34 @@ zLib License	| `zlib`
 Open Data Commons Public Domain Dedication and License | `pddl`
 Lesser General Public License For Linguistic Resources | `lgpl-lr`
 
+## Why is it useful to calculate the carbon emissions of my model?
+
+Training ML models is often energy intensive can produce a substantial carbon footprint, as described by [Strubell et al.](https://arxiv.org/abs/1906.02243). It's therefore important to *track* and *report* the emissions of models to get a better idea of the environmental impacts of our field.
+
+
+## What information should I include about the carbon footprint of my model?
+
+If you can, include information about:
+- where the model was trained (in terms of location)
+- the hardware that was used -- e.g. GPU, TPU or CPU, and how many
+- training type: pre-training or fine-tuning
+- the estimated carbon footprint of the model, calculated in real-time with the [Code Carbpn](https://github.com/mlco2/codecarbon) package or after the fact using the [ML CO2 Calculator](https://mlco2.github.io/impact/).
+
+## Carbon footprint metadata
+
+The structure of the metadata should be:
+
+```yaml
+---
+co2_eq_emissions:
+  - emissions: "in grams of CO2"
+  - source: "source of the information, either directly from autonlp, code carbon or from a scientific article documenting the model"
+  - training_type: "pretraining or fine-tuning"
+  - geographical location: "as granular as possible, for instance Quebec, Canada or Brooklyn, NY, USA"
+  - hardware used: "how much compute and what kind, e.g. 8 v100 GPUs"
```

Comment:
```suggestion
co2_eq_emissions:
  emissions: "in grams of CO2"
  source: "source of the information, either directly from autonlp, code carbon or from a scientific article documenting the model"
  training_type: "pretraining or fine-tuning"
  geographical_location: "as granular as possible, for instance Quebec, Canada or Brooklyn, NY, USA"
  hardware_used: "how much compute and what kind, e.g. 8 v100 GPUs"
```

in yaml if adding dashes you're defining an object that contains an array of objects that all have a single key (see screenshot)
<img width="2140" alt="Screenshot 2021-12-10 at 20 41 33" src="https://user-images.githubusercontent.com/326577/145632279-5e3a7da2-fd30-407f-ba1e-c839bbc9c0b4.png">

whereas you want just a single object

Reply:
hmm, this wasn't fixed, unless I'm mistaken. Can you fix it in another PR, @sashavor? Thanks!

`co2_eq_emissions` should be a dict, not an array of single-key dicts...

Reply:
i.e. you need to remove the dashes and make sure that key names are single words with underscores. 🙏